src/TimePicker.jsx

```diff
@@ -108,40 +108,51 @@ export default class TimePicker extends PureComponent {
 
   renderInputs() {
     const {
+      amPmAriaLabel,
       clearIcon,
+      clearAriaLabel,
       clockIcon,
+      clockAriaLabel,
       disableClock,
       disabled,
       format,
+      hourAriaLabel,
       isOpen,
       locale,
       maxDetail,
       maxTime,
+      minAriaLabel,
       minTime,
       name,
+      nativeInputAriaLabel,
       required,
       value,
     } = this.props;
 
     return (
       <div className={`${baseClassName}__wrapper`}>
         <TimeInput
+          amPmAriaLabel={amPmAriaLabel}
           className={`${baseClassName}__inputGroup`}
           disabled={disabled}
           format={format}
           locale={locale}
+          hourAriaLabel={hourAriaLabel}
           isClockOpen={isOpen}
           maxDetail={maxDetail}
           maxTime={maxTime}
+          minAriaLabel={minAriaLabel}
           minTime={minTime}
           name={name}
+          nativeInputAriaLabel={nativeInputAriaLabel}
           onChange={this.onChange}
           placeholder={this.placeholder}
           required={required}
           value={value}
         />
         {clearIcon !== null && (
           <button
+            aria-label={clearAriaLabel !== null ? clearAriaLabel : undefined}
```

Comment:
Why not simply 
```suggestion
            aria-label={clearAriaLabel}
```
? I think `null` is fine and would simply cause `aria-label` not to be added?

Reply:
You're right. I read somewhere it needed to be undefined, but I just tested it and it worked fine.